### PR TITLE
Fix overlapping Search Results

### PR DESCRIPTION
**Pull request by schwartz-concordium**
## Purpose

Due to introduction of new tab ‘Contract’ the search result columns was overlapping. This PR fixes this.
Bug
- https://concordium.atlassian.net/browse/CBW-1364

## Changes

- Change Search Result View from going to full screen at screen size LG (1024) instead of XL (1280).
- Change navigation tabs from going to burger style at screen size less than XL (1280) instead of first at LG (1024)

## Checklist

- [x] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: frontend/src/components/Search/Search.vue [hunk 1]
```diff
@@ -259,9 +264,11 @@ import ContractLink from '~/components/molecules/ContractLink.vue'
 import { useDateNow } from '~/composables/useDateNow'
 import type { Position } from '~/composables/useTooltip'
 import NodeLink from '~/components/molecules/NodeLink.vue'
+import { Breakpoint } from '~~/src/composables/useBreakpoint'
```
Comment (gitguillebetis) [inline frontend/src/components/Search/Search.vue:267]: is it needed?

File: frontend/src/components/Search/Search.vue [hunk 2]
```diff
@@ -259,9 +264,11 @@ import ContractLink from '~/components/molecules/ContractLink.vue'
...
 
 const { NOW } = useDateNow()
 const drawer = useDrawer()
+const { breakpoint } = useBreakpoint()
```
Comment (gitguillebetis) [inline frontend/src/components/Search/Search.vue:271]: is it needed?

Comment (schwartz-concordium) [reply frontend/src/components/Search/Search.vue]: no I have removed it - nice catch :-)

File: frontend/src/components/Search/Search.vue (hunk 1 above)
Comment (schwartz-concordium) [reply frontend/src/components/Search/Search.vue]: no I have removed it - nice catch :-)

**Review by gitguillebetis — APPROVED** [review]

**Pull request merged**